openquake/server/views.py

```diff
@@ -1264,6 +1263,75 @@ def web_engine_get_outputs_aelo(request, calc_id, **kwargs):
                        warnings=warnings))
 
 
+def get_aggrisk(calc_id):
```

Comment:
The docstring should say something like "Converting the aggrisk dataframe into a table like this:
..." and add an example table.

Reply:
Done